package.json

```diff
@@ -87,7 +87,7 @@
     "semver": "5.5.1",
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/61374056-1449e200-a8a4-11e9-915a-fa3f02b5f6dd.png)

Reply:
updated both markup table and data table to use the same css